source/scpp/src/scp/QuorumSetUtils.cpp

```diff
@@ -134,14 +134,14 @@ namespace
 //      { t:1, { innerSet } } into innerSet
 
 void
-normalizeQSetSimplify(SCPQuorumSet& qSet, NodeID const* idToRemove)
+normalizeQSetSimplify(SCPQuorumSet& qSet, bool remove, NodeID const idToRemove)
```

Comment:
I don't see why those changes are necessary either ?

Reply:
Previously, the default value of `idToRemove` was `nullptr`, so it can be used to determine whether to remove the node or not. But we can not call this function in the situation where there's no node to be removed because the type `NodeID` is changed into `ulong`.

Reply:
Why not ? It passes a point to a `NodeID`, I don't see what's different here ?